guides/v2.2/rest/search-endpoint.md

```diff
@@ -0,0 +1,83 @@
+---
+group: rest-api
+title: Search with the /search endpoint
+contributor_name: 
+contributor_link: 
+---
+
+By using `GET {{URL}}rest/V1/search`, you can search through the catalog. The obtained results will come from your configured search engine (**Stores** > Settings > **Configuration** > **Catalog** > **Catalog Search** > **Search engine**).
+This means you can collect results from Elasticsearch, for example, that are very similar to the results that are visible by using the regular frontend search.
```

Comment:
I would not be strict here to which engine is used even as example, maybe better to write
"This means you can collect results from search engine that is enabled. And those results will be equal to the results that are coming by using the regular frontend search."